parsl/tests/test_sites/test_ec2/ec2.py

```diff
@@ -1,91 +0,0 @@
-ec2OnDemand = {
```

Comment:
is this EC2 stuff relevant to the PR?

Reply:
It's not I keep pushing these changes to the wrong branch instead of master. While not related, can we keep it so that it'll get to master ?